Taarafo.Core/Brokers/Storages/IStorageBroker.GroupPosts.cs

```diff
@@ -11,5 +11,6 @@ namespace Taarafo.Core.Brokers.Storages
     public partial interface IStorageBroker
     {
         ValueTask<GroupPost> DeleteGroupPostAsync(GroupPost groupPost);
+        ValueTask<GroupPost> InsertGroupPostAsync(GroupPost groupPost);
```

Comment:
Methods should be in order of CRUD operations. So, it should be 
`InsertGroupPostAsync()`
`DeleteGroupPostAsync()`